docs/api.json

```diff
@@ -0,0 +1,1070 @@
+          "auth": null,
+          "scope": [],
+          "body": {
+            "organisationId": "Number",
+            "name": "String",
+            "gender": "String",
+            "birthYear": "String",
+            "email": "String",
+            "phoneNumber": "String",
+            "postCode": "String",
+            "emailConsent": "Boolean",
+            "smsConsent": "Boolean",
+            "invitationCode": "String"
+          },
+          "response": "types.adminUser"
+        }
+      },
+      "/login/visitor": {
+        "POST": {
+          "auth": true,
+          "intendedFor": ["cbStandardLogin"],
+          "scope": ["users_details-own:write"],
+          "body": {
+            "secret": "String"
+          },
+          "response": "types.visitorUser"
+        }
+      },
+      "/login": {
+        "POST": {
+          "auth": null,
+          "scope": null,
+          "body": {
+            "userName": "String",
+            "secret": "String"
+          },
+          "response": {}
+        }
+      },
+      "/:id": {
+        "GET": {
+          "description": "Retrieve details of a specific user",
+          "auth": true,
+          "intendedFor": ["twineAdmin", "fundingBody", "cbAdmin"],
+          "scope": ["users_details-children:read"],
+          "query": {
+            "fields": ["String"]
+          },
+          "response": "types.visitorUser || types.volunteerUser || types.adminUser"
+        },
+        "PUT": {
+        "description": "Update details of a specific user",
+        "intendedFor": ["twineAdmin", "fundingBody", "cbAdmin"],
+        "scope": ["users_details-children:write"],
+        "body": {
+          "name": "String",
+          "regionId": "Number",
+          "sectorId": "Number",
+          "organisationId": "Number",
+          "gender": "String",
+          "birthYear": "String",
+          "email": "String",
+          "phoneNumber": "String",
+          "emailConsent": "Boolean",
+          "smsConsent": "Boolean",
+          "qrCode": "String"
+        },
+        "response": "types.visitorUser || types.volunteerUser || types.adminUser"
+        },
+        "DELETE": {
+        "description": "Delete a specific user",
+        "intendedFor": ["twineAdmin", "fundingBody", "cbAdmin"],
+        "scope": ["users_details-children:write"],
+        "body": {
+          "userId": "Number"
+        },
+        "response": null
+        }
+      },
+      "/password/reset": {
+        "POST": {
+          "description": "Reset password for a single user",
+          "auth": true,
+          "intendedFor": ["twineAdmin", "fundingBody", "cbAdmin", "visitor", "volunteer", "volunteerAdmin"],
+          "scope": ["user_details-own:write"],
+          "body": {
+            "userId": "Number",
+            "email": "String",
+            "password": "String",
+            "confirmPassword": "String"
+          },
+          "response": null
+        }
+      },
+      "/admin/me": {
```

Comment:
Why is this nested under `/users`? Could it not live under a top-level `/admins` and fulfill the same purpose?